app/views/thesis/process_theses.html.erb

```diff
@@ -45,8 +45,18 @@
           --
           Status: <span class="thesis-status text-<%= thesis.css_alert_type %>">
             <%= thesis.status %></span>
-          <%# Other stuff should go here when e.g. people can mark as done or
-              a download link exists. %>
+
+          --
+
+          <br />
+          Files:  <% if thesis.files.attached? %>
+                    <% thesis.files.each do |file| %>
+                      <br />
```

Comment:
Is the linebreak going to look weird given that we're in a panel footer?

Reply:
It seems fine for now... but also I did open a ticket to clean this up because it's awful.

https://mitlibraries.atlassian.net/browse/TI-51

Reply:
👍

Reply:
FWIW you can see what it looks like if people for some reason upload a bajillion files: https://library-thesis-dropbox-s-pr-55.herokuapp.com/process